code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -881,3 +881,20 @@
 		to_chat(M, "<span class = 'notice'>[pick("Diamond skies where white deer fly.","Sipping strawberry tea.","Silver raindrops drift through timeless, Neverending June.","Crystal ... pearls free, with love!","Beaming love into me.")]</span>")
 	..()
 	. = 1
+
+/datum/reagent/consumable/catnip_tea
+	name = "Catnip Tea"
+	id = "catnip_tea"
+	description = "A sleepy and tasty catnip tea!"
+	color = "#101000" // rgb: 16, 16, 0
+	nutriment_factor = 0
+	taste_description = "sugar and catnip"
+	glass_icon_state = "teaglass"
+	glass_name = "glass of catnip tea"
+	glass_desc = "A purfect drink for a cat."
+
+/datum/reagent/consumable/catnip_tea/on_mob_life(mob/living/carbon/M)
+	M.emote("nya")
+	to_chat(M, "<span class = 'notice'>[pick("Headpats feel nice.", "Backrubs would be nice.", "Mew")]</span>")
+	M.adjustArousalLoss(5)
+	M.adjustStaminaLoss(3)
```

Comment:
Kev please comment on this.

Reply:
@kevinz000 I forgot I can do this. Specifically stamina loss.

Reply:
sleepy tea!